### DISCUSSION TITLE
How to see all inactive and active containers?

### DISCUSSION THREAD

**Discussion by Luxvao** [containerd/nerdctl#1078] [Q&A] [answered]
I'm new to nerdctl and I can't find a way to list all containers. I don't know if I'm even understanding it correctly, but if I run nerdctl run -it --name debian debian it should create a container with the name debian, right? Even if that's not how you create a container I also tried nerdctl container create --name debian debian. Can you only see the container whilst there running and not when you stop them? Can I run containers that I made and installed a couple of tools just by going nerdctl run <container_name>?
Thanks in advance

**Comment by fahedouch** [comment, marked as answer]
@Luxvao  `nerdctl ps -a` fit your need ?

**Reply by Luxvao** [reply]
Yes it seems to be working. Thanks

**Reply by Luxvao** [reply]
But how do I start a container? If I run nerdctl run -it <container_name> or nerdctl run -it <container_id> it just tries to pull the image with that name.
Thanks in advance

**Reply by fahedouch** [reply]
you mean ` nerdctl run -it <image_name> ` or `nerdctl run -it <image_id>`  ? ; `nerdctl run`will fetch image and then create container under the hood
